### DISCUSSION TITLE
Can only ssh into the camera when I scan the QR Code in the app. it lasts for ~ 3 minutes

### DISCUSSION THREAD

**Discussion by kronkinatorix** [gtxaspec/wz_mini_hacks#691] [Q&A]
the app says the connection times out but the camera is clearly connected to my home network because it has an ip and I can ssh into it. 
It says it's timed out about 3 minutes later and drops the connection.

Any guidance here would be appreciated. If there's a way to manually set up networking and then never have the app touch my phone again that would be ideal. 

Coincidentally, I'm open to putting into some time into getting some actual documentation for this camera together, along with setup instructions that don't stop before configuration. Since I can't actually get this setup myself I'm not sure how I'd do this, but I believe in projects like this and rather than bitching about how terrible the guidance on this repo is I'm willing to do something about it. I lack the know-how, but if I can spare anyone else the frustration getting this fucking thing to work has caused me I'll have added some good to the world

**Comment by virmaior** [comment]
some clarification here:
1. `the app` = Wyze's app?
2. did you set the camera up using the wyze method?
3. does the camera have internet access?
4. Is the /configs path empty?

**Comment by kronkinatorix** [comment]
Hey. 
I returned the cameras. not worth the headache. For prosperity's sake though:

1. yes. 
2. I can't. See the post above. I generate the qr code in the app, the camera reads the qr code, it says either "connecting" or "cannot find network name", connects to  the network for about three minutes, then I get a "connection timed out" from the camera. 
It did not work on my home router/modem in gateway mode. It did not work with my Android phone as a hotspot using my home internet or using my cellular data. It did not work with the same on a spare iphone I have. It did not work with my other pc acting as a hotspot. It did not work with my router in bridge mode and either of my 2 PCs acting as an access point. It didn't work using the wifi of the pizza place across the road from me.
4. Yes. Which works for the ~3 minute window I have before it "times out"
5. yep.
